webapp/components/interop-dashboard.js

```diff
@@ -672,5 +708,74 @@ class InteropDashboard extends PolymerElement {
     this.$.toggleStable.setAttribute('aria-pressed', true);
     this.$.toggleExperimental.setAttribute('aria-pressed', false);
   }
+
+  getSortIcon(index) {
+    if (this.toggleStates[index]) {
+      return '/static/expand_more.svg';
+    }
+    return '/static/expand_less.svg';
+  }
+
+  sortRows(rows, index, tableState) {
+    if(index === 0) {
+      const sortedFeatureOrder = [];
+
+      // Alphabetize by focus area column
+      if(tableState.sortColumn === -1) {
+        const rowNames = [];
+        for(let i = 0; i < rows.length; i++) {
+          const feature = rows[i];
+          rowNames[i] = [feature, this.getRowInfo(feature, 'description')];
+        }
+        rowNames.sort((a, b) => {
+          a[1] - b[1];
+        });
```

Comment:
Something is wrong here, because technically this sort function is not returning any value (no return statement). Oddly enough, it seems like the table is being sorted correctly, but I think that is because the values for the interop tables are already sorted alphabetically in `interop-data.js`, and this sort call has no effect.